src/Events/Migration/v0.38/01-update-function.sql

```diff
@@ -0,0 +1,47 @@
+DROP FUNCTION IF EXISTS events.getappevents(character varying, character varying, timestamp with time zone, timestamp with time zone, text[], text[], integer);
```

Comment:
This is the function currently in use? If so, we will have requests that fail. Rather drop it in a later release when we are sure no pods are using the function.

Reply:
Gjort